plugins/convertPathData.js

```diff
@@ -167,6 +170,14 @@ exports.fn = (root, params) => {
             (computedStyle['stroke-linecap'].type === 'dynamic' ||
               computedStyle['stroke-linecap'].value !== 'butt');
           const maybeHasStrokeAndLinecap = maybeHasStroke && maybeHasLinecap;
+          // TODO: when stroke is used, prefer z more
+          // Z is the same as going home at 90 and 0 degrees, but not at other degrees
```

Comment:
```suggestion
          // Z is the same as going home at 90 and 0 degrees, but not at other degrees
```

Can we drop the TODO?
When or if we have a solution for preferring `z` in more cases involving stroke, we can create an issue and work on it. I'd prefer not to pollute the code with tasks, though.

If you already have an idea of some cases we could prefer `z` but currently don't, I'd appreciate it if you could open an issue with the scenarios and I can look at it if you don't have time.

Reply:
I removed the whole thing, given that since I implemented always converting LINE-z to z, this isn't as much of a concern as it was when I first wrote the comment